src/components/loadable.cjsx

```diff
@@ -21,14 +23,17 @@ module.exports = React.createClass
     renderError: React.PropTypes.func.isRequired
     renderBug: React.PropTypes.func.isRequired
 
+  getInitialState: ->
+    refreshButton: false
+
   getDefaultProps: ->
 
     # Enables a renderStatus prop function with a component other than a div
-    renderLoading: ->
-      <div className='-loading'>Loading...</div>
+    renderLoading: (refreshButton) ->
```

Comment:
How would this work with Components with a custom `renderLoading` method (I think the calendar may have this)?

It seems like to make sure a refresh button is always available either:

1. all `render*` method must always take a `refreshButton` argument (not a func)
2. render will always append a `refreshButton` for loading and error states